src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -283,13 +289,14 @@ void remove_virtual_functionst::remove_virtual_function(
       it->source_location.set_comment(comment);
   }
 
-  goto_programt::targett next_target=target;
-  next_target++;
-
   goto_program.destructive_insert(next_target, new_code);
 
   // finally, kill original invocation
   target->make_skip();
+
+  remove_skip(goto_program, target, next_target);
```

Comment:
Here we are removing skips from the virtual function block? Consider commenting that's what the `[target, next_target]` range corresponds to

Reply:
Done.